src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -457,11 +465,54 @@ static Status deserializeTensor(const std::string& requestedName, const KFSReque
     return StatusCode::OK;
 }
 
+typedef std::unique_ptr<OvmsPyTensor> OvmsPyTensorPtr;
+
+#if (PYTHON_DISABLE == 0)
+static Status deserializeTensor(const std::string& requestedName, const KFSRequest& request, std::unique_ptr<PyObjectWrapper<py::object>>& outTensor, PythonBackend * pythonBackend) {
+    auto requestInputItr = request.inputs().begin();
+    auto status = getRequestInput(requestInputItr, requestedName, request);
+    if (!status.ok()) {
+        return status;
+    }
+    auto inputIndex = requestInputItr - request.inputs().begin();
+    auto& bufferLocation = request.raw_input_contents().at(inputIndex);
+    try {
+        std::vector<py::ssize_t> shape;
+        for (int i = 0; i < requestInputItr->shape().size(); i++) {
+            if (requestInputItr->shape()[i] < 0) {
```

Comment:
We should include test for 0 dimensions and 0 in dimension value

Reply:
Sure

Reply:
precision: FP32,  data:[6.0f], shape: [] -> scalar
precision: FP32, data: [], shape: [1, 32, 32, 0, 1] -> no data
models should be ready in our src/test dir